sample-apps/java/client/devserver-client-sample/src/main/java/timefetch/TimeCallerApp.java

```diff
@@ -0,0 +1,42 @@
+package timefetch;
+
+import java.io.BufferedOutputStream;
+import java.io.PrintStream;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.client.WorkflowStub;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class TimeCallerApp {
+    static {
```

Comment:
See my comments elsewhere about lines 12-16 and the comment on line 19 (since the same code appears in one or more other files)

Reply:
Thank you again.